packages/pulumi-aws/src/apps/gateway/GatewayApi.ts

```diff
@@ -0,0 +1,165 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+
+import { ApplicationContext, loadGatewayConfig, PulumiApp } from "@webiny/pulumi-sdk";
+import { applyCustomDomain, CustomDomainParams } from "../customDomain";
+
+export interface GatewayApiConfig {
```

Comment:
BTW was thinking a bit more about this file and the question I now have is - will this file actually be deleted completely once WCP is in place?

Reply:
No, why? 
This is a pulumi code for API gateway - to split traffic between different deployed API variants.